js/integration-tests/solidity-tests/test/testContext.ts

```diff
@@ -59,20 +65,35 @@ export class TestContext {
       L1_CHAIN_TYPE,
       l1SolidityTestRunnerFactory()
     );
+    await context.edrContext.registerSolidityTestRunnerFactory(
+      OP_CHAIN_TYPE,
+      opSolidityTestRunnerFactory()
+    );
 
     return context;
   }
 
-  defaultConfig(): SolidityTestRunnerConfigArgs {
+  defaultConfig(
+    chainType: string = L1_CHAIN_TYPE
+  ): SolidityTestRunnerConfigArgs {
+    let localPredeploys = undefined;
+    if (chainType === L1_CHAIN_TYPE) {
```

Comment:
I'm not sure whether this should be accounted for in these tests, but just to make sure we're not forgetting anything: the generic chain type also uses `l1GenesisState`